admin/linux/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Distro with Qt 5.6
```

Comment:
There should be a README comment stating what this docker image is for.

(Also a license?)